README.md

```diff
@@ -265,6 +265,27 @@ replicator.start();
 
 Read more in [the replication docs](https://github.com/cloudant/sync-android/blob/master/doc/replication.md).
 
+#### Authentication
+
+Sync-android uses session cookies by default to authenticate with the server if
+credentials are provided with in the URL. If you need more fine grain control
```

Comment:
thinking about this a little bit more, maybe we shouldn't mention proxy here, since we mention it later with for proxy authentication, perhaps we should say

> If you want more fine-grained control over authentication, provide an interceptor to perform the authentication for each request. For example, if you are using middleware such as Envoy....